### DISCUSSION TITLE
Support for open source gf180mcu-pdk

### DISCUSSION THREAD

**Discussion by m-usama-z** [ucb-bar/hammer#721] [Development]
Are there any plans to support the following open source PDK in hammer?
https://github.com/google/gf180mcu-pdk
Many thanks so long as there's an intention to do so, even if not soon.

**Comment by harrisonliew** [comment]
We have an intention, but currently no manpower here to do so. We'd love to have external contributors make a plugin!

**Comment by m-usama-z** [comment]
I'm severely lacking in experience to be able to do this properly (never used a real PDK except sky130), but the MPWs of gf180mcu are a valuable opportunity for (almost) poor people like me, so I'll keep it in mind :)
